classes/install.yaml

```diff
@@ -116,22 +110,19 @@ packageScript: |
     # Only copy shared libraries.
     installPackageLib()
     {
-        if [[ ${INSTALL_SHARED:-} ]] ; then
-            installCopy "$@" /usr/ /usr/lib/ \
-                "/usr/lib/*.so.*" \
-                "/usr/lib/lib*-*.so" \
-                "!*"
-            installStripAll .
-        fi
+        installCopy "$@" /usr/ \
+            /usr/lib/ "/usr/lib/*.so*" \
+            /usr/bin/ "/usr/bin/*.dll" "/usr/bin/*.pdb" \
+            "!*"
+        installStripAll .
     }
 
     # Everything except shared or static libraries or header files.
     installPackageBin()
     {
-        installCopy "$@" '!/usr/include/' \
-            "!/usr/lib/*.a" "!/usr/lib/*.so*"  \
-            "!/usr/lib/pkgconfig" \
-            "!/usr/lib/cmake"
+        installCopy "$@" /usr/ \
+            /usr/bin/ "/usr/bin/***" \
+            "!*"
```

Comment:
What's the reason to change this? Now the function does not install anything except binaries. This will miss e.g. plugins, configuration files, ...

Reply:
I have to re-check this.